app/models/manageiq/providers/autosde/storage_manager/cloud_volume_snapshot.rb

```diff
@@ -3,14 +3,37 @@ class ManageIQ::Providers::Autosde::StorageManager::CloudVolumeSnapshot < ::Clou
   supports :delete
 
   def self.raw_create_snapshot(ext_management_system, options = {})
-    pass
```

Comment:
Just noticed this, there is no `pass` keyword in ruby this will throw an exception